src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -192,8 +192,8 @@ private void ProcessStarting(InteractiveHostOptions options)
             var metadataService = _workspace.CurrentSolution.Services.MetadataService;
             ImmutableArray<string> referencePaths;
 
-            // reset configuration:
-            if (File.Exists(_responseFilePath))
+            // reset configuration (null initialization file indicates "noconfig"):
+            if (options.InitializationFile != null && File.Exists(_responseFilePath))
```

Comment:
My understanding is that ```options.InitializationFile``` is set (indirectly?) from ```_responseFilePath```.  It's the value we hand off to the server.  It would probably be more transparent to pass the evaluator a flag indicating ```noconfig``` but the information was already available.